crates/storage/provider/src/providers/blockchain_provider.rs

```diff
@@ -2348,8 +2354,7 @@ mod tests {
             &mut rng,
             TEST_BLOCKS_COUNT,
             TEST_BLOCKS_COUNT,
-            None,
-            None,
+            BlockParams::default(),
```

Comment:
@shekhirin seems like the provider test_receipt_provider_* are failing on here. it looked to me like they are passing cos both lists are empty, debugging..